__tests__/api-tests/stakeholder-api.spec.js

```diff
@@ -4,11 +4,11 @@ const { expect } = require('chai');
 const server = require('../../server/app');
 const { stakeholderOne, stakeholderTwo } = require('./seed-data-creation');
 
-describe('Stakeholder API tests.', () => {
+describe.skip('Stakeholder API tests.', () => {
   describe('Stakeholder GET', () => {
     it(`Should raise validation error with error code 422 -- 'limit' query parameter should be an integer  `, function (done) {
       request(server)
-        .get(`/stakeholder`)
+        .get(`/`)
```

Comment:
Actually this API pack should be 'stakeholders'.  The mount point might be 'stakeholder' as well, but the type of the resources should always been the name of the route in our APIs